docs/setup/install.md

```diff
@@ -0,0 +1,34 @@
+# Installation
+
+**Requires Kubernetes 1.22 or newer**. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
```

Comment:
Add a `Before you begin` and a `Requirements` section.

Maybe we can expand that you can use kubernetes 1.21 if you enable the SuspendJob [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)

Reply:
umm Requirements and `before you begin` are kind of the same section, no?

Reply:
`Before you begin` is more about what you need to know to follow the tutorial.

But sure, since both just have one item, one `Before you begin` section is enough.